srcpkgs/lua54-luarepl/template

```diff
@@ -17,37 +18,40 @@ _rocksrel=1
 _package() {
 	luarocks --lua-version="$1" --tree="${PKGDESTDIR}/usr/" \
 		make --deps-mode=none --no-manifest "${_rockname}-${version}-${_rocksrel}.rockspec"
-		echo /destdir/${pkgname}-${version}
-		sed -i -e "s|/destdir/${pkgname}-${version}||" "${PKGDESTDIR}/usr/bin/rep.lua"
+		if [ "$CROSS_BUILD" ]; then
+			vsed -i "${PKGDESTDIR}/usr/bin/rep.lua" -e "s|/destdir/${XBPS_CROSS_TRIPLET}/${pkgname}-${version}||g"
+		else
+			vsed -i "${PKGDESTDIR}/usr/bin/rep.lua" -e "s|/destdir/${pkgname}-${version}||g"
+		fi
```

Comment:
```suggestion
		vsed -i "${PKGDESTDIR}/usr/bin/rep.lua" -e "s|${PKGDESTDIR}||g"
```

Reply:
Thanks! I had to wrap the PKGDESTDIR variable in a `realpath` invocation to normalise it and remove the double slashes. But otherwise this is much simpler and readable! I'll update #53798 as well